Android/app/src/main/java/io/github/project_travel_mate/mytrips/MyTripInfoActivity.java

```diff
@@ -628,17 +633,19 @@ public void onResponse(Call call, final Response response) throws IOException {
                                 addNewFriend.setVisibility(GONE);
                                 friendEmail.setVisibility(GONE);
                                 noFriendTitle.setVisibility(VISIBLE);
-                                noFriendTitle.setTypeface(null, Typeface.BOLD);
+//                                noFriendTitle.setTypeface(null, Typeface.BOLD);
```

Comment:
don't comment. if you want to remove, remove it

Reply:
Done

Reply:
not done. It's still here